tests/cross/src/test/scala/tests/pc/CompletionOverrideSuite.scala

```diff
@@ -942,11 +1137,24 @@ class CompletionOverrideSuite extends BaseCompletionSuite {
        |class Under extends Over {
        |  def inner: Outer.Inner = ${0:???}
        |}
-       |""".stripMargin
+       |""".stripMargin,
+    compat = Map(
+      "3" ->
+        """|trait Over {
+           |  object Outer {
+           |    class Inner
+           |  }
+           |  def inner: Outer.Inner
+           |}
+           |class Under extends Over {
+           |  def inner: Under#Outer.Inner = ${0:???}
```

Comment:
It should be `Outer.Inner` (without `Under#`), but I'm not sure how can we make it 🤔 Can we work on this in another issue?
`asSeenFrom(<Under>)` should `Outer.Inner`, instead of `Under#Outer.Inner`...

Reply:
Sure!